app/templates/base.html

```diff
@@ -0,0 +1,71 @@
+{% extends 'govuk_frontend_jinja/template.html' %}
+
+{% set assetPath = url_for('static', filename='').rstrip('/') %}
+
+{% block pageTitle %}DLUHC Funding Service Design Iteration 1{% endblock %}
+
+{% block head %}
+  <meta name="description" content="{{ service_meta_description }}">
+  <meta name="keywords" content="{{ service_meta_keywords }}">
+  <meta name="author" content="{{ service_meta_author }}">
+  <!--[if gt IE 8]><!--><link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='govuk-frontend-4.0.0.min.css') }}" /><!--<![endif]-->
```

Comment:
I would personally like to extend the max line length. I think that the pep 8 default of 79 is too restrictive, and one can easily change this in a config.

Reply:
We have decided on 120 characters.

Reply:
120 may be too long. @harryyo @ramsharma-prog what is your opinion? The default of 79 is nice and keeps everything very pythonic, especially if we use black for formatting since it will nicely break apart stuff.

Reply:
@SentientHousePlant  I think 120 is better than 79

Reply:
79 has a nice history and many massive projects seem to cope with it...

My thoughts

**79**
- Industry standard for python.
- Pep8 compliance, no need for reconfiguration.
- encourages pythonic coding practices
- my portrait monitor.
**longer / 120**
- less line splitting, but perhaps just write shorter lines?
- better support for longer variable/function names
- uses wide screen monitors more efficiently (dev dependent)
- may encourage writing "magic" lines which are long but impossible to fathom.

Reply:
okay let's stick with 79!

Reply:
Returned to PEP8 with removal of .flake8 config file